tests/FunctionalTests/ConversationTest.php

```diff
@@ -145,4 +144,35 @@ public function testAll()
         $this->assertSame($conversationId, $message->getConversationId());
         $this->assertNull($message->getAttachment());
     }
+
+    public function testShouldUpdateConversation()
```

Comment:
Is this test really useful as we're already updating a conversation in the previous one?

Reply:
Yes you are right! I removed it, and I also added more properties to assert when we update the conversation in the test above.